components/scream/cmake/machine-files/summit-cpu.cmake

```diff
@@ -0,0 +1,6 @@
+# Load all kokkos settings from Ekat's mach file
+set (EKAT_MACH_FILES_PATH ${CMAKE_CURRENT_LIST_DIR}/../../../../externals/ekat/cmake/machine-files)
+include (${EKAT_MACH_FILES_PATH}/kokkos/serial.cmake)
```

Comment:
Is there any OpenMP CPU support in EKAT?

Reply:
yes

Reply:
Also, we should probably create a `p9.cmake` file in EKAT, and then include that here. Those files are simply enabling the specified arch in kokkos, which ultimately allows to add arch-specific optimization flags. We could directly do that here, to be fair. Doing it in a file in EKAT only allows to change a single file (in EKAT) if (or maybe _when_) kokkos changes their CMake vars names/logic.

Reply:
Let's do it in an additional PR for EKAT